File: Lab1/Lab1/lab1j/src/db/PowerSupplyRepository.java
package db;

import dto.PowerSupplyDto;

import java.util.LinkedList;
import java.util.List;

class PowerSupplyRepository
    {
        public static List<PowerSupplyDto> findAll()
        {
            List<PowerSupplyDto>  powerSupplies = new LinkedList<>();
            powerSupplies.add(new PowerSupplyDto(300, -2));
            powerSupplies.add(new PowerSupplyDto(300, 2));
            return powerSupplies;
        }
    }

File: WpfApp1/db/PowerSupplyRepository.cs
using Lab1.dto;
using Lab1Code.model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab1.db
{
    class PowerSupplyRepository : IRepository<PowerSupplyDto>
    {
        private MyDbContext myDbContext;

        public PowerSupplyRepository(MyDbContext myDbContext)
        {
            this.myDbContext = myDbContext;
        }

        List<PowerSupplyDto> IRepository<PowerSupplyDto>.findAll()
        {
            //List<PowerSupplyDto>  powerSupplies = new List<PowerSupplyDto>();
            //powerSupplies.Add(new PowerSupplyDto(300, -2));
            //powerSupplies.Add(new PowerSupplyDto(300, 2));
            //myDbContext.powerSupplyDtos.AddRange(powerSupplies);
            //myDbContext.SaveChanges();
            return myDbContext.powerSupplyDtos.ToList();
        }
    }
}
